pkg/storage/digest.go

```diff
@@ -0,0 +1,10 @@
+package storage
+
+import remoteexecution "github.com/bazelbuild/remote-apis/build/bazel/remote/execution/v2"
+
+// EmptyDigest is a REv2 Digest representing an object of size 0 hashed
+// with SHA256
+var EmptyDigest = &remoteexecution.Digest{
```

Comment:
Does this not exist within any of our imports? Buildbarn etc

Reply:
No, because the pkg/digest API is written in such a way that it supports multiple hashing algorithms; not just SHA-256. A singleton instance for SHA-256 wouldn't be helpful.